tests/test_router_bgp.rb

```diff
@@ -624,6 +624,40 @@ def test_routerbgp_get_reconnect_interval_default
     bgp.destroy
   end
 
+  def test_routerbgp_set_get_route_distinguisher
+    asnum = 55
+    vrf = 'blue'
+    bgp = RouterBgp.new(asnum, vrf)
```

Comment:
You're not using asnum or vrf anywhere else in this method so just get rid of the variables and put the values explicitly in the call:

`   bgp = RouterBgp.new(55, 'blue')